doc-content/jbpm-docs/src/main/asciidoc/Kafka/integration-kafka-con.adoc

```diff
@@ -9,12 +9,12 @@ ifdef::PAM,DM[]
 endif::PAM,DM[]
 is a streaming platform. It acts as a message broker, passing messages, which are sorted into topics, between applications in a software environment.
 
-Using {PRODUCT}, you can create business processes that send and receive Kafka messages in the following ways:
+Using {PRODUCT}, you can create business processes that send and receive Kafka messages. You can use the following methods to create such business processes:
 
-* Create a start event, intermediate catch event, or boundary event (attached to a human task) of the type `message`. {KIE_SERVER} automatically subscribes to the Kafka topic that is defined in the message. A message triggers the event. The event node acts as the consumer of the message and can pass the content of the message to the subsequent node in the process.
+* Create a start event, intermediate catch event, or boundary event (attached to a human task) of the `message` type. {KIE_SERVER} automatically subscribes to the Kafka topic that is defined in the message. A message triggers the event. The event node acts as the consumer of the message and can pass the content of the message to the subsequent node in the process.
 
 * Create an end event or intermediate throw event of the type `message`. When the process triggers the event, {KIE_SERVER} sends a Kafka message in the topic that is defined in the message. The message contains the data that is configured in the event. The event node acts as the producer of the message.
 
 * Add the `KafkaPublishMessages` custom task to the process. This task does not require the {KIE_SERVER} Kafka capability but can be more complicated to configure than message events.
 
-* Configure your service and {KIE_SERVER} to emit Kafka messages about every completed process, case, and task when transactions are committed.
+* Configure your service and {KIE_SERVER} to automatically post (emit) Kafka messages about every completed process, case, and task when transactions are committed.
```

Comment:
[question] - Is there a style rule for adding alternative words such as "emit" in parentheses?

Reply:
[Style] Having reread the section, maybe replacing "post" and "emit" with "send":

s/to automatically post (emit) Kafka messages/to automatically send Kafka messages

Reply:
I have now fixed this issue by getting rid of "emit" but adding "configure with an emitter" instead.